docs/how-to-install.md

```diff
@@ -30,20 +30,28 @@ The Gravitino binary distribution package contains the following files:
-    |   └── lakehouse-iceberg/      # Apache Iceberg catalog dependencies and configurations.
-    |   └── jdbc-mysql/             # JDBC MySQL catalog dependencies and configurations.
-    |   └── jdbc-postgresql/        # JDBC PostgreSQL catalog dependencies and configurations.
-    |── conf/                       # All configurations for Gravitino.
-    |   ├── gravitino.conf          # Gravitino server configuration.
-    |   ├── gravitino-env.sh        # Environment variables, etc., JAVA_HOME, GRAVITINO_HOME, and more.
-    |   └── log4j2.properties       # log4j configuration for the Gravitino server.
-    |── libs/                       # Gravitino server dependencies libraries.
-    |── logs/                       # Gravitino server logs. Automatically created after the Gravitino server starts.
-    |── data/                       # Default directory for the Gravitino server to store data.
-    └── scripts/                    # Extra scripts for Gravitino.
+    |   └── hadoop/                             # Apache Hadoop catalog dependencies and configurations.
+    |   └── hive/                               # Apache Hive catalog dependencies and configurations.
+    |   └── jdbc-doris/                         # JDBC doris catalog dependencies and configurations.
+    |   └── jdbc-mysql/                         # JDBC MySQL catalog dependencies and configurations.
+    |   └── jdbc-postgresql/                    # JDBC PostgreSQL catalog dependencies and configurations.
+    |   └── kafka/                              # Apache Kafka PostgreSQL catalog dependencies and configurations.
+    |   └── lakehouse-iceberg/                  # Apache Iceberg catalog dependencies and configurations.
+    |   └── lakehouse-paimon/                   # Apache Paimon catalog dependencies and configurations.
+    |── conf/                                   # All configurations for Gravitino.
+    |   ├── gravitino.conf                      # Gravitino server and Gravitino Iceberg REST server configuration.
+    |   ├── gravitino-iceberg-rest-server.conf  # Gravitino server configuration.
+    |   ├── gravitino-env.sh                    # Environment variables, etc., JAVA_HOME, GRAVITINO_HOME, and more.
+    |   └── log4j2.properties                   # log4j configuration for the Gravitino server and Gravitino Iceberg REST server.
+    |── libs/                                   # Gravitino server dependencies libraries.
+    |── logs/                                   # Gravitino server and Gravitino Iceberg REST server logs. Automatically created after the server starts.
+    |── data/                                   # Default directory for the Gravitino server to store data.
+    |── iceberg-rest-server/                    # Gravitino Iceberg REST server package and dependencies libraries.
+    └── scripts/                                # Extra scripts for Gravitino.
 ```
```

Comment:
I think it's not enough, you should have separate paragraphs to describe how to start, stop, etc.

Reply:
Why you don't address this comment?

Reply:
I'm still thinking about how to organize it more user friendly.